examples/aws/ingress/values.yaml

```diff
@@ -0,0 +1,32 @@
+# This example shows how to deploy and expose GraphDB with AWS Load Balancer Controller Ingress.
+
+configuration:
+  externalUrl: graphdb-example-dns-name.com # Change this to your ALB DNS name or Route53 if you use it.
+
+service:
+  enabled: true
+  type: NodePort
+  port: 7200
+
+ingress:
+  enabled: true
+  annotations:
+    ingressClassName: "alb"
+    alb.ingress.kubernetes.io/load-balancer-name: graphdb-ingress
+    alb.ingress.kubernetes.io/target-type: ip
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}]'
```

Comment:
I wonder if 443 example would be helpful.

Reply:
I will add one.